tests/unit/fem/test_estimator.cpp

```diff
@@ -221,7 +223,9 @@ TEST_CASE("Least-squares ZZ estimator on 3D NCMesh", "[NCMesh]")
 {
    // Setup
    const auto order = GENERATE(2, 3);
-   Mesh mesh = Mesh::MakeCartesian3D(2, 2, 2, Element::HEXAHEDRON);
+   // const auto element = GENERATE(Element::TETRAHEDRON, Element::HEXAHEDRON);
```

Comment:
Delete the debris or reenable for all element types (preferable).

Reply:
The estimator is broken for Tets, so I will leave only the Hex test for now.